operators/src/error.rs

```diff
@@ -135,9 +136,23 @@ pub enum Error {
         source: chrono::format::ParseError,
     },
 
+    LoadingInfo {
+        reason: String, // cannot nest error because it's from service crate
```

Comment:
Box<dyn Error>? But there were some errors doing this, right?

Reply:
Maybe something like?
```
    LoadingInfo {
        reason: Box<dyn std::error::Error + Send + Sync>, // cannot nest error because it's from service crate
    },
```
but I can't really test it right now because the service crate where this error shall be user does not yet use it because I splitted the PRs. Shall I leave it like this or remove the error since it's not yet used.

Reply:
I would at least make a TODO there with this solution. But I guess if you manage to use the Box then it will work in the future.

Reply:
yes just add a todo

Reply:
3ef00f432b46f288cdcee28a2d2bc23f01f520c5